test/run_pass/issue/issue-133.js

```diff
@@ -13,6 +13,14 @@
  * limitations under the License.
  */
 
+if (this.driver) {
+  var originalConsole = console;
+  console = require('tools/commonJS/modules/console_wrapper');
```

Comment:
Is this `commonJS` related to [this CommonJS](http://www.commonjs.org/)? If not, it would be better to use other name to avoid confusion.